<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Template:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QComboBox" name="templateComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="wordlistGroupBox">
      <property name="title">
       <string>Page:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;b&gt;Title:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pageTitleLE">
         <property name="text">
          <string>Page title</string>
         </property>
         <property name="placeholderText">
          <string>Page title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;b&gt;Page HTML:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="pageHtmlLE">
           <property name="placeholderText">
            <string>Page HTML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pageHtmlSelectB">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;b&gt;Page JS:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="pageJsLE">
           <property name="placeholderText">
            <string>Page JS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pageJsPB">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;b&gt;Page CSS:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="pageCssLE">
           <property name="placeholderText">
            <string>Page CSS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pageCssPB">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="generateB">
      <property name="toolTip">
       <string>StartStop</string>
      </property>
      <property name="text">
       <string>GENERATE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTemplate">
    <property name="title">
     <string>Template</string>
    </property>
    <addaction name="actionTemplateAdd"/>
    <addaction name="actionTemplateRemove"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuTemplate"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTemplateAdd">
   <property name="text">
    <string>Add new</string>
   </property>
  </action>
  <action name="actionTemplateRemove">
   <property name="text">
    <string>Remove selected</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
